README_ADMIN.md

```diff
@@ -0,0 +1,218 @@
+a tool meant for administrators of RStudio Connect.  `rsconnect-admin` allows administrators
+to search for content and initiate content builds during a server migration.
+
+The `rsconnect-admin` CLI uses the same server "nicknames" as the `rsconnect` CLI.
+See [this section](./README.md#managing-server-information) of the main README
+for details on adding a new RStudio Connect server.
+
+
+### Search
+
+The following are some examples of how an administrator might use the
+`rsconnect-admin content search` subcommand to search for content on RStudio Connect.
+This could be useful for administrators to identify content that needs to be
+rebuilt during a server migration. See [this section](#building-content) for details
+on when a content build might be necessary.
+
+By default, the `rsconnect-admin content search` command will return metadata for ALL
+of the content on a RStudio Connect server, both published and unpublished content.
+
+```bash
+# only published content
+$ rsconnect-admin content search --published
+
+# only unpublished content
+$ rsconnect-admin content search --unpublished
+
+# published content where the python version is at least 3.9.0
+$ rsconnect-admin content search --published --py-version ">=3.9.0"
+
+# published content where the R version is exactly 3.6.3
+$ rsconnect-admin content search --published --r-version "==3.6.3"
+
+# published content where the content type is a static RMD
+$ rsconnect-admin content search --content-type rmd-static
+
+# published content where the content type is either shiny OR fast-api
+$ rsconnect-admin content search --content-type shiny --content-type python-fastapi
+
+# any content, published or unpublished, where the title contains the text "Stock Report"
+$ rsconnect-admin content search --title-contains "Stock Report"
+
+# published content, results are ordered by when the content was last deployed
+$ rsconnect-admin content search --published --order-by last_deployed
+
+# published content, results are ordered by when the content was created
+$ rsconnect-admin content search --published --order-by created
+```
+
+
+### Building Content
+
+RStudio Connect caches R and Python packages in the configured
+[`Server.DataDir`](https://docs.rstudio.com/connect/admin/appendix/configuration/#Server.DataDir).
+Sometimes, these package caches become stale and need to be rebuilt. This refresh
+occurs automatically when an RStudio Connect user visits the content via the
+RStudio Connect dashboard. Some content, however, is high priority or not accessed
+frequently via the dashboard (API content, emailed reports). In these cases, administrators
+can preemptively build specific content items using the `rsconnect-admin build` subcommands.
+This way the user does not have to pay the build cost when the content is accessed next.
+
+The following are some common scenarios where performing a content build might be necessary:
```

Comment:
It feels like there is an overlap between the paragraph above and these reasons to rebuild. Perhaps you can narrow the paragraph above down a little? I've wondered if the order should switch, if that would be clearer to have what the common scenarios might be before you explain the technical details of the caching / automatic building.